src/Compilers/CSharp/Portable/CodeGen/EmitExpression.cs

```diff
@@ -2645,6 +2652,12 @@ private void EmitSizeOfExpression(BoundSizeOfOperator boundSizeOfOperator)
             EmitSymbolToken(type, boundSizeOfOperator.SourceType.Syntax);
         }
 
+        private void EmitMethodTokenExpression(BoundMethodToken node)
+        {
+            _builder.EmitOpCode(ILOpCode.LoadMethodTokenAsInteger_i4);
```

Comment:
I think it would be easier to debug if we used `-1 * pseudotoken` but, otherwise, that sounds reasonable.